src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Authentication.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace System.Net.Http.Functional.Tests
 {
-    [ActiveIssue(29802, TargetFrameworkMonikers.Uap)]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Tests would need to be rewritten due to behavior differences with WinRT")]
```

Comment:
Do we need a workitem for this?

Reply:
I don't think we will schedule this work. So, I didn't want to create a tracking item for it.